### PR TITLE
[PROTOCOL-78] Audit: Increase Code Coverage

### PR DESCRIPTION
**Pull request by sypeer**
It includes:

- New unit tests to increase code coverage.

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (salazarguille) [inline .vscode/settings.json:1]: I think we don't need this file. Each dev uses a different configuration for the VSCode (or use another IDE)

File: test/base/EtherCollateralLoansGetBorrowerLoansTest.js [hunk 2]
```diff
@@ -107,6 +107,10 @@ contract('EtherCollateralLoansCreateLoanWithTermsTest', function (accounts) {
 
             const loan = await instance.loans.call(mockLoanIDCounter)
             
+            const borrowedLoans = await instance.getBorrowerLoans(borrowerAddress);
```
Comment (salazarguille) [inline test/base/EtherCollateralLoansGetBorrowerLoansTest.js:110]: I think if you want to create unit tests for the getBorrowerLoans function, it is better if you create a new unit test file  (EtherCollateralLoansGetBorrowerLoansTest).

We don't want to change a test for another function. Also, remember the filename and the name at the top of the file (contract('EtherCollateralLoansCreateLoanWithTermsTest') )have to match.

File: test/base/LoanTermsConsensusProcessRequestTest.js [hunk 3]
```diff
@@ -167,5 +170,79 @@ contract('LoanTermsConsensusProcessRequestTest', function (accounts) {
                 assert.equal(error.reason, expectedErrorMessage);
             }
         })
-    })
+    });
+
+    withData({
+        _9_nonce_taken: [
+            1, 320, [responseOne], true, 'REQUEST_NONCE_TAKEN'
```
Comment (salazarguille) [inline test/base/LoanTermsConsensusProcessRequestTest.js:177]: I think this scenario is already covered on the branch develop here https://github.com/teller-protocol/teller-protocol-v1/blob/develop/test/base/LoanTermsConsensusProcessRequestTest.js#L108
Please review it is covering the same scenario.

Comment (salazarguille) [reply test/base/LoanTermsConsensusProcessRequestTest.js:201]: Fixed

File: test/base/EtherCollateralLoansGetBorrowerLoansTest.js (hunk 2 above)
Comment (salazarguille) [reply test/base/EtherCollateralLoansGetBorrowerLoansTest.js:91]: Fixed

File: .vscode/settings.json (hunk 1 above)
Comment (salazarguille) [reply .vscode/settings.json:1]: Fixed

File: .solcover.js [hunk 4]
```diff
@@ -5,5 +5,6 @@ module.exports = {
     compileCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle compile --network coverage',
     skipFiles: [
         'mock/',
+        'providers/openzeppelin/SignedSafeMath.sol' // It is not included due to it is a OpenZeppelin contract.
```
Comment (alex-murashkin) [inline .solcover.js:8]: 👍

**Pull request merged**